pkg/resources/prometheusRules.go

```diff
@@ -68,6 +68,13 @@ func (r *AlertReconcilerImpl) ReconcileAlerts(ctx context.Context, client k8scli
 		}
 	}
 
+	//removing RHOAMUpgradeExpectedDuration30minExceeded alert as it was renamed (30min -> 60min)
+	alertToDelete := AlertConfiguration{
+		AlertName: "RHOAMUpgradeExpectedDuration30minExceeded",
+		Namespace: "openshift-monitoring",
```

Comment:
Sorry Valery just noticed now..can we use here observability.OpenshiftMonitoringNamespace instead of having a string

Reply:
Thanks @MStokluska , I actually tried do it initially, but have some very strange error, so used hardcoded. I will try again, and if not work, I will share the error

Reply:
Actually, that would be an import cycle; since this is a temporary code let's go with it